doc/chapters/api/index.rst

```diff
@@ -24,88 +13,276 @@ Methods
 
 Ping
 ----
-Ping()
+Controls availability of com.pelagicore.SoftwareContainerAgent interface
+
+:Parameters:
+        - **None**
+
+:Return Value:
+        - **None** 
+
+:Example Usage:
+
+::
+
+        dbus-send --system --print-reply \
+        --dest=com.pelagicore.SoftwareContainerAgent \
+        /com/pelagicore/SoftwareContainerAgent \
+        com.pelagicore.SoftwareContainerAgent.Ping
```

Comment:
I thought the comment was about why the command is split into four parts. I don't think it's necessary to explain that. However, if we will use dbus-send in the examples etc. then I agree it's a good idea to point out what the opath and iface is etc. But i think that's better suited then in the Getting started (or whatever that turns out to be in the end) rather than here. See my comment about consolidating with the rest of the docs.

Kursat, why have example invocations here as well as in the Getting started?